cooker/cooker.py

```diff
@@ -444,10 +446,24 @@ def update_directory(self, method, local_dir, has_remote, branch, rev):
     def generate(self):
         info('Generating dirs for all build-configurations')
 
+        self.read_local_conf_version()
+
         for build in BuildConfiguration.ALL.values():
             if build.buildable():
                 self.prepare_build_directory(build)
 
+    def read_local_conf_version(self):
+        try:
+            self.distro.local_conf_version = "{}".format(self.distro.DEFAULT_CONF_VERSION)
```

Comment:
We should not change the distro-class here. `local_conf_version` should be in `self`.

There is no need to do the format-kludge to be sure it is a string we can do the following:

```suggestion
            self.local_conf_version = str(self.distro.DEFAULT_CONF_VERSION)
```